jetstream/test/kv_test.go

```diff
@@ -1213,6 +1213,35 @@ func TestListKeyValueStores(t *testing.T) {
 }
 
 func TestKeyValueMirrorCrossDomains(t *testing.T) {
+	keyExists := func(t *testing.T, kv jetstream.KeyValue, key string, expected string) jetstream.KeyValueEntry {
+		var e jetstream.KeyValueEntry
+		var err error
+		checkFor(t, 5*time.Second, 10*time.Millisecond, func() error {
```

Comment:
If we will hit very slow/busy worker, maybe it's worth to have all max timeouts set to 10 seconds.